js/src/tooltip.js

```diff
@@ -636,6 +620,20 @@ const Tooltip = (($) => {
       return config
     }
 
+    _cleanTipClass() {
+      const $tip = $(this.getTipElement())
+      const tabClass = $tip.attr('class').match(BSCLS_PREFIX_REGEX)
+      if (tabClass !== null && tabClass.length > 0) {
+        $tip.removeClass(tabClass.join(''))
+      }
+    }
+
+    _handlePopperPlacementChange(data) {
+      if (data.originalPlacement !== data.placement) {
```

Comment:
this `if` should be removed because it omits update if popper.js places tooltip again to default position.
Then data.orginalPlacement === data.placement but we changed class before.

@FezVrasta 
What is purpose of `data.flipped` flag?
According to my tests it's equivalent to `data.originalPlacement !== data.placement` but it will be more valuable if `data.flipped` will store information when "flip" happens during current update.

@Johann-S 
the same change should go for popover (onCreate, onUpdate)